### ISSUE TITLE
MVC not rendering on xml design view, messed up

### ISSUE THREAD

**Issue by almazangc** [prolificinteractive/material-calendarview#1135]
![image](https://user-images.githubusercontent.com/59057047/206867943-82c0291c-9f0c-4ab8-8838-0844548368d1.png)
Please refer below

**Comment by almazangc** [comment]
<com.prolificinteractive.materialcalendarview.MaterialCalendarView
                android:id="@+id/adapter_analytic_parent_item_calendar"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_below="@+id/adapter_analytic_parent_item_date_started"
                app:mcv_selectionColor="@color/A80_BRIGHT_SKY_BLUE"
                app:mcv_selectionMode="range"
                app:mcv_showOtherDates="all" />

**Comment by vivekgupta4Git** [comment]
When you want to work on design just downgrade the library version to 1.6.0 but upgrade it when you run the app.

**Comment by almazangc** [comment]
> When you want to work on design just downgrade the library version to 1.6.0 but upgrade it when you run the app.

is there no fix aside from rolling back version?

**Comment by vivekgupta4Git** [comment]
> > When you want to work on design just downgrade the library version to 1.6.0 but upgrade it when you run the app.
> 
> is there no fix aside from rolling back version?

yes ! download the source code and inside the constructor of MaterailCalendarView put this line of code
```

  if(isInEditMode())
    {
      AndroidThreeTen.init(context);
    }
```
build the project and include it in your working project

**Comment by almazangc** [comment]
> > > When you want to work on design just downgrade the library version to 1.6.0 but upgrade it when you run the app.
> > 
> > 
> > is there no fix aside from rolling back version?
> 
> yes ! download the source code and inside the constructor of MaterailCalendarView put this line of code
> 
> ```
> 
>   if(isInEditMode())
>     {
>       AndroidThreeTen.init(context);
>     }
> ```
> 
> build the project and include it in your working project

Would that mean a lot of work, like forking, and reposting the project for dependency usage?
